api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -34,18 +34,125 @@
 #include "tensor_filter_single.h"
 
 /**
- * @brief Default time to wait for an output in appsink (3 seconds).
+ * @brief Default time to wait for an output in appsink (10 seconds).
+ * @note unittests take over 5sec to complete
```

Comment:
plz add @todo tag to update the API description (single header and java) when the APIs are changed to single-new.

Reply:
Updated back to 3sec
This issue is being resolved as a different PR in the tensorflow build